code/modules/clothing/neck/mantles.dm

```diff
@@ -0,0 +1,63 @@
+//Mantles!
+
+/obj/item/clothing/neck/mantle
+	name = "mantle"
+	desc = "A heavy quilted mantle, for keeping your shoulders warm and stylish."
+	icon_state = "mantle"
+	item_state = "mantle"
```

Comment:
The inhand sprite always defaults to `icon_state` if `item_state` isn't set, so since they're all the same in this file you can just remove them.
```suggestion
	icon_state = "mantle"
```